animalai/play.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+from pathlib import Path
+
+from animalai import AnimalAIEnvironment, arenas
+from animalai.executable import find_executable
+
+def play(configuration_file: str = None, env_path: str = None) -> None:
+    """
+    Load a config file and play
+    :param configuration_file: str path to a yaml configuration. Plays animalai.arenas.GoodGoal_Random.yml by default
+    :param env_path: str path to AAI environment executable. Looks for it in root by default.
+    :return: None
+    """
+
+    print("Initializing AAI environment")
+    environment = AnimalAIEnvironment(
+        file_name=env_path if env_path is not None else str(find_executable(Path(""))),
+        base_port=5005 + random.randint(0, 1000), # use a random port to avoid problems if a previous version exits slowly
```

Comment:
I think we can get rid of this comment as its a bit redundant and the code itself is self-explanatory and most people would not be bothered (my personal opinion).